Tools/MonoGame.Content.Builder.Editor/Common/PipelineController.cs

```diff
@@ -123,6 +129,26 @@ public static PipelineController Create(IView view)
             return new PipelineController(view);
         }
 
+        public void SetupProjectFileWatcher(string projectFilePath)
```

Comment:
You are not uninitializing FileWatcher when you close the project (`CloseProject` method).

Reply:
Fixed